app/src/main/res/layout/activity_main.xml

```diff
@@ -21,6 +21,8 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:id="@+id/bottom_navigation"
+        app:itemTextAppearanceActive="@style/Widget.BottomNavigationView"
```

Comment:
Since this is probably an attempt to assign a custom style, this should be done with `style="@style/Widget.BottomNavigationView"`. Why not simply using `app:fontFamily`?

Reply:
It didn't work with `app:fontFamily`

Reply:
True, I double checked it. I looked it up, and we may as well change the font for the whole app globally in the style files as specified [here](https://github.com/material-components/material-components-android/blob/master/docs/theming/Typography.md#customization). I'm gonna approve this pull request but leave #44 open.